modules/virt-customizing-storage-profile.adoc

```diff
@@ -67,3 +67,42 @@ status:
 <2> The `volumeMode` that you select.
 +
 After you save your changes, the selected values appear in the storage profile `status` element.
+
+== Setting a default cloning strategy using a storage profile
+
+You can use storage profiles to set a default cloning method for a storage class, creating a _cloning strategy_. Setting cloning strategies can be helpful, for example, if your storage vendor only supports certain cloning methods. It also allows you to select a method that limits resource usage or maximizes performance.
+
+Cloning strategies can be specified by setting the `cloneStrategy` attribute in a storage profile to one of these values:
+
+* `snapshot` - This method will be used by default when snapshots are configured. This cloning strategy uses a temporary volume snapshot to clone the volume. The storage provisioner must support CSI snapshots.
+* `copy` - This method uses a source Pod and a target Pod to copy data from the source volume to the target volume. Host-assisted cloning is the least efficient method of cloning.
```

Comment:
s/Pod/pod/

x2

for these [general references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references).

Reply:
Thanks. I was unsure if that had changed and we are not referring to the object.